config/industrialforegoing.cfg

```diff
@@ -31,19 +31,19 @@ general {
             S:0_poor=0
 
             #  [default: 4000000]
```

Comment:
Why did you change the Infinity Drill?

Reply:
Because energy system in E2E is reworked, and some things like Mek reactors have huge output. In this environment, when drill become available you already have power production enough to reach maximum level in instant (10 minutes at 200 kRF/t).
Its not "Fix". Just suggestion.